build.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
Why did you delete this script?
It's used in our CI pipeline which is now broken https://circleci.com/gh/hypriot/qemu-register/5

Reply:
Ooops, I didn't wan't break your CI 😅.

I removed build.sh, because I mean that construct `docker build -t hypriot/qemu-register:local .` (or another, written manual) better, because user must to understand what he  is doing (what he is building). Also I documented that in README.md.

One moment I fixed in CI.